sense-android-library/test/java/nl/sense_os/datastorageengine/CSUtils.java

```diff
@@ -0,0 +1,320 @@
+package nl.sense_os.datastorageengine;
+
+import org.apache.http.conn.ssl.SSLSocketFactory;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.ProtocolException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.zip.GZIPOutputStream;
+
+import javax.net.ssl.HttpsURLConnection;
+
+import nl.sense_os.datastorageengine.CommonSenseProxy;
+
+/**
+ * Created by ronald on 24-8-15.
+ */
+public class CSUtils {
```

Comment:
@excaliburnan @RonaldOlsthoorn If the purpose of this class is a utils for testing, then It's better to but it in a test folder